girder_worker/docker/io/girder.py

```diff
@@ -0,0 +1,26 @@
+import six
+
+from . import StreamReader
+
+
+class GirderFileStreamReader(StreamReader):
```

Comment:
Same question here -- how specific is this functionality to the docker runtime mode? Should it be reorganized so it's not namespaced within the `docker` package?

Reply:
Belongs in docker package, at least for the moment.